addons/account/models/account_report.py

```diff
@@ -448,11 +448,18 @@ class AccountReportExpression(models.Model):
              " in case the carryover destination requires more complex logic."
     )
 
-    _sql_constraints = [(
-        "domain_engine_subformula_required",
-        "CHECK(engine != 'domain' OR subformula IS NOT NULL)",
-        "Expressions using 'domain' engine should all have a subformula."
-    )]
+    _sql_constraints = [
+        (
+            "domain_engine_subformula_required",
+            "CHECK(engine != 'domain' OR subformula IS NOT NULL)",
+            "Expressions using 'domain' engine should all have a subformula."
+        ),
+        (
+            "line_label_uniq",
+            "UNIQUE(report_line_id,label)",
+            "The expression label must be unique per report line !"
```

Comment:
Globally speaking, we tend to avoid using exclamation marks in error messages ; they make them a little to aggressive ^^